src/app/families/FamilyDeliveries.ts

```diff
@@ -92,6 +92,9 @@ export class FamilyDeliveries extends IdEntity {
     distributionCenter = new DistributionCenterId(this.context, {
         allowApiUpdate: Roles.admin
     });
+    isDistCenterInactive() {
+        return true; // this.distributionCenter.isNotActive;
```

Comment:
@noam-honig : how do I get to this field from here?

Reply:
It depends on the use case. Why do you need it here?

Reply:
this.context.for(DistributioCenter).lookup(this.DistributionCenter).IsNotActive.Value